packages/bindgen/spec.yml

```diff
@@ -381,6 +381,11 @@ records:
         type: bool
         default: false
       sync_config: Nullable<std::shared_ptr<SyncConfig>>
+      # Used internally as a flag for opening a synced realm locally.
+      # (Needed for TS when returning a binding.RealmConfig_relaxed.)
```

Comment:
I probably should have made the comment a little bit clearer, but the `force_sync_history` is used for our internal `transformConfig()` in `Realm.ts`:
```ts
/**@internal */
public static transformConfig(config: Configuration): {
  schemaExtras: RealmSchemaExtra;
  bindingConfig: binding.RealmConfig_Relaxed;
} {
  // ...
  return {
    schemaExtras,
    bindingConfig: {
      // ...
      forceSyncHistory: config.openSyncedRealmLocally,
    },
  };
}
```

I believe `force_sync_history` is [needed by Core](https://github.com/realm/realm-core/blob/b503314b43aa91db9c300d8dff83292e0d251757/src/realm/object-store/shared_realm.hpp#L158-L160) for this.

Reply:
Plus, the spec.yml is intended to be SDK-agnostic since it will be used by many. I know we aren't perfect about that now, but as we are getting closer to other SDKs using it, we should try to make an effort to maintain separation of concerns.

Reply:
👍